cmd/config.go

```diff
@@ -305,18 +305,16 @@ func addPathsFromDirectory(ctx context.Context, stderr io.Writer, a *appState, d
 }
 
 // ConfigToWrapper converts the Config struct into a ConfigOutputWrapper struct
-func ConfigToWrapper(config *Config) *ConfigOutputWrapper {
-	cfgw := &ConfigOutputWrapper{Global: config.Global, Paths: config.Paths}
-	var providers []*ProviderConfigWrapper
+func (config *Config) ConfigToWrapper() *ConfigOutputWrapper {
```

Comment:
The word `Config` in the method name is redundant. This could be named just `ToWrapper` or `AsWrapper` or maybe even `Wrapped`.